examples/jf-control-app/commands/pairing/PairingCommand.h

```diff
@@ -106,6 +106,7 @@ class PairingCommand : public CHIPCommand,
             break;
         case PairingMode::Code:
             AddArgument("anchor", 0, 1, &mAnchor);
+            AddArgument("execute-jcm", 0, 1, &mExecuteJCM);
```

Comment:
Can we move the "execute-jcm" and "anchor" arguments into the PairingCommand class constructor, since they're common to all subcommands?

Reply:
Thanks for this suggestion. Pushed the fix, pls check.